### PR TITLE
add sidebar to basic page layout

### PR DESCRIPTION
**Pull request by janno42**
adds a sidebar

### CODE REVIEW

File: _1327/static/less/1327.less [hunk 1]
```diff
@@ -150,3 +150,12 @@ footer p {
 	overflow: hidden;
 	background-color: @gray-lighter;
 }
+
+.content {
+	padding-left: 30px;
```
Comment (invliD) [inline _1327/static/less/1327.less]: Use Bootstrap variables instead.

File: _1327/static/less/1327.less [hunk 2]
```diff
@@ -150,3 +150,12 @@ footer p {
...
+}
+
+.sidebar {
+	border-left: 1px solid @gray;
+	padding-right: 20px;
```
Comment (invliD) [inline _1327/static/less/1327.less]: See above.

**Comment by janno42** [comment]
now using variables

**Comment by invliD** [comment]
+1

**Comment by Bartzi** [comment]
:+1: after Travis worked...

**Pull request merged**